src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -248,6 +248,17 @@ export const validateAddress = (
     // and only one confirmed address came back from the API
     const showModal = showAddressValidationModal(suggestedAddresses);
 
+    // push data to dataLayer for analytics
+    window.dataLayer.push({
```

Comment:
We should probably use the `recordEvent` helper for a little bit of safety. Would be a super quick change to make.

https://github.com/department-of-veterans-affairs/vets-website/blob/c72006cf2836bfbdf5895d28d9cb011d7a35dd03/src/platform/monitoring/record-event.js#L8